src/Components/CreateImageWizard/formComponents/RepositoriesStatus.tsx

```diff
@@ -13,9 +13,14 @@ import { ApiRepositoryResponse } from '../../../store/contentSourcesApi';
 type RepositoryStatusProps = {
   repoStatus: ApiRepositoryResponse['status'];
   repoUrl: ApiRepositoryResponse['url'];
+  Name: ApiRepositoryResponse['name'];
 };
 
-const RepositoriesStatus = ({ repoStatus, repoUrl }: RepositoryStatusProps) => {
+const RepositoriesStatus = ({
+  repoStatus,
+  repoUrl,
+  Name,
```

Comment:
For consistency, I would change this to `repoName`

Reply:
I removed the name from this file, solve it in a different way